### ISSUE TITLE
PermissionError when building perl-module-build

### ISSUE THREAD

**Issue by timothymillar** [bioconda/bioconda-recipes#8522]
Running into a permissions error when trying to build recipe `perl-module-build` (needed as a prerequisite for another package).
Same issue encountered when building `perl-module-build` individually or as a requirement for another package.

The offending file:
```
$ ls -lh /workspace/cfltxm/miniconda3/envs/biocondabuilder/conda-bld/perl-module-build_1522805102380/_h_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placeh/bin/config_data
-r-xr-xr-x. 1 cfltxm powerplant 7.3K Apr  4 13:26 /workspace/cfltxm/miniconda3/envs/biocondabuilder/conda-bld/perl-module-build_1522805102380/_h_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placeh/bin/config_data
(biocondabuilder) [cfltxm@aklppf31 recipes]$ 

```

There was a minor warning during the build which i think is un-related:
```
Checking optional features...
inc_bundling_support....disabled
  requires:
    ! inc::latest is not installed

ERRORS/WARNINGS FOUND IN PREREQUISITES.  You may wish to install the versions
of the modules indicated above before proceeding with this installation
```

Full details bellow, any help is appreciated.

conda info:
```
$ conda info

     active environment : biocondabuilder
    active env location : /workspace/cfltxm/miniconda3/envs/biocondabuilder
            shell level : 2
       user config file : /home/cfltxm/.condarc
 populated config files : 
          conda version : 4.5.0
    conda-build version : 3.8.0
         python version : 3.6.4.final.0
       base environment : /workspace/cfltxm/miniconda3  (writable)
           channel URLs : https://repo.anaconda.com/pkgs/main/linux-64
                          https://repo.anaconda.com/pkgs/main/noarch
                          https://repo.anaconda.com/pkgs/free/linux-64
                          https://repo.anaconda.com/pkgs/free/noarch
                          https://repo.anaconda.com/pkgs/r/linux-64
                          https://repo.anaconda.com/pkgs/r/noarch
                          https://repo.anaconda.com/pkgs/pro/linux-64
                          https://repo.anaconda.com/pkgs/pro/noarch
          package cache : /workspace/cfltxm/miniconda3/pkgs
                          /home/cfltxm/.conda/pkgs
       envs directories : /workspace/cfltxm/miniconda3/envs
                          /home/cfltxm/.conda/envs
               platform : linux-64
             user-agent : conda/4.5.0 requests/2.18.4 CPython/3.6.4 Linux/3.10.0-693.11.6.el7.x86_64 centos/7 glibc/2.17
                UID:GID : 2395:2395
             netrc file : None
           offline mode : False
```

conda-build:
```
$ conda-build perl-module-build
Adding in variants from internal_defaults
INFO:conda_build.variants:Adding in variants from internal_defaults
Attempting to finalize metadata for perl-module-build
INFO:conda_build.metadata:Attempting to finalize metadata for perl-module-build
Solving environment: ...working... done
Solving environment: ...working... done
BUILD START: ['perl-module-build-0.4224-pl526_0.tar.bz2']
Solving environment: ...working... done
Solving environment: ...working... done

## Package Plan ##

  environment location: /workspace/cfltxm/miniconda3/envs/biocondabuilder/conda-bld/perl-module-build_1522805102380/_h_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placeh


The following NEW packages will be INSTALLED:

    cloog:        0.18.0-0         
    gcc:          4.8.5-7          
    gmp:          6.1.2-h6c8ec71_1 
    isl:          0.12.2-0         
    libgcc-ng:    7.2.0-hdf63c60_3 
    libstdcxx-ng: 7.2.0-hdf63c60_3 
    mpc:          1.0.3-hec55b23_5 
    mpfr:         3.1.5-h11a74b3_2 
    perl:         5.26.0-hae598fd_0

Preparing transaction: ...working... done
Verifying transaction: ...working... done
Executing transaction: ...working... done
Source cache directory is: /workspace/cfltxm/miniconda3/envs/biocondabuilder/conda-bld/src_cache
Found source in cache: Module-Build-0.4224_b74c2f6e84.tar.gz
Extracting download
source tree in: /workspace/cfltxm/miniconda3/envs/biocondabuilder/conda-bld/perl-module-build_1522805102380/work

Checking optional features...
inc_bundling_support....disabled
  requires:
    ! inc::latest is not installed

ERRORS/WARNINGS FOUND IN PREREQUISITES.  You may wish to install the versions
of the modules indicated above before proceeding with this installation

Created MYMETA.yml and MYMETA.json
Creating new 'Build' script for 'Module-Build' version '0.4224'
Building Module-Build
t/00-compile.t ................. ok     
t/PL_files.t ................... ok   
t/actions/installdeps.t ........ ok   
t/actions/manifest_skip.t ...... ok   
t/add_property.t ............... ok     
t/add_property_array.t ......... ok   
t/add_property_hash.t .......... ok   
t/basic.t ...................... ok     
t/bundle_inc.t ................. skipped: $ENV{MB_TEST_EXPERIMENTAL} is not set
t/compat.t ..................... ok     
t/compat/exit.t ................ ok   
t/debug.t ...................... ok   
t/destinations.t ............... ok       
t/ext.t ........................ ok       
t/extend.t ..................... ok     
t/files.t ...................... ok   
t/help.t ....................... ok     
t/install.t .................... ok     
t/install_extra_target.t ....... ok   
t/manifypods.t ................. ok     
t/manifypods_with_utf8.t ....... ok   
t/metadata.t ................... ok     
t/metadata2.t .................. ok     
t/mymeta.t ..................... ok     
t/new_from_context.t ........... ok   
t/notes.t ...................... ok     
t/par.t ........................ skipped: No compiler found
t/parents.t .................... ok     
t/perl_mb_opt.t ................ ok   
t/pod_parser.t ................. ok     
t/ppm.t ........................ skipped: No compiler found
t/properties/dist_suffix.t ..... ok   
t/properties/license.t ......... ok   
t/properties/module_name.t ..... ok   
t/properties/needs_compiler.t .. ok     
t/properties/release_status.t .. ok    
t/properties/requires.t ........ ok   
t/properties/share_dir.t ....... ok     
t/resume.t ..................... ok   
t/runthrough.t ................. ok     
t/sample.t ..................... ok   
t/script_dist.t ................ ok   
t/signature.t .................. ok     
t/test_file_exts.t ............. ok   
t/test_reqs.t .................. ok   
t/test_type.t .................. ok   
t/test_types.t ................. ok     
t/tilde.t ...................... ok     
t/unit_run_test_harness.t ...... ok   
t/use_tap_harness.t ............ ok   
t/versions.t ................... ok   
t/write_default_maniskip.t ..... ok   
t/xs.t ......................... skipped: No compiler found
All tests successful.
Files=53, Tests=1133, 54 wallclock secs ( 0.26 usr  0.10 sys + 36.41 cusr 11.30 csys = 48.07 CPU)
Result: PASS
Building Module-Build
Installing /powerplant/workspace/cfltxm/miniconda3/envs/biocondabuilder/conda-bld/perl-module-build_1522805102380/_h_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placeh/man/man1/config_data.1
Installing /powerplant/workspace/cfltxm/miniconda3/envs/biocondabuilder/conda-bld/perl-module-build_1522805102380/_h_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placeh/lib/site_perl/5.26.0/Module/Build.pm
Installing /powerplant/workspace/cfltxm/miniconda3/envs/biocondabuilder/conda-bld/perl-module-build_1522805102380/_h_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placeh/lib/site_perl/5.26.0/Module/Build/Base.pm
Installing /powerplant/workspace/cfltxm/miniconda3/envs/biocondabuilder/conda-bld/perl-module-build_1522805102380/_h_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placeh/lib/site_perl/5.26.0/Module/Build/Notes.pm
Installing /powerplant/workspace/cfltxm/miniconda3/envs/biocondabuilder/conda-bld/perl-module-build_1522805102380/_h_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placeh/lib/site_perl/5.26.0/Module/Build/ConfigData.pm
Installing /powerplant/workspace/cfltxm/miniconda3/envs/biocondabuilder/conda-bld/perl-module-build_1522805102380/_h_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placeh/lib/site_perl/5.26.0/Module/Build/Authoring.pod
Installing /powerplant/workspace/cfltxm/miniconda3/envs/biocondabuilder/conda-bld/perl-module-build_1522805102380/_h_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placeh/lib/site_perl/5.26.0/Module/Build/Config.pm
Installing /powerplant/workspace/cfltxm/miniconda3/envs/biocondabuilder/conda-bld/perl-module-build_1522805102380/_h_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placeh/lib/site_perl/5.26.0/Module/Build/PPMMaker.pm
Installing /powerplant/workspace/cfltxm/miniconda3/envs/biocondabuilder/conda-bld/perl-module-build_1522805102380/_h_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placeh/lib/site_perl/5.26.0/Module/Build/Compat.pm
Installing /powerplant/workspace/cfltxm/miniconda3/envs/biocondabuilder/conda-bld/perl-module-build_1522805102380/_h_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placeh/lib/site_perl/5.26.0/Module/Build/Dumper.pm
Installing /powerplant/workspace/cfltxm/miniconda3/envs/biocondabuilder/conda-bld/perl-module-build_1522805102380/_h_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placeh/lib/site_perl/5.26.0/Module/Build/API.pod
Installing /powerplant/workspace/cfltxm/miniconda3/envs/biocondabuilder/conda-bld/perl-module-build_1522805102380/_h_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placeh/lib/site_perl/5.26.0/Module/Build/Cookbook.pm
Installing /powerplant/workspace/cfltxm/miniconda3/envs/biocondabuilder/conda-bld/perl-module-build_1522805102380/_h_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placeh/lib/site_perl/5.26.0/Module/Build/PodParser.pm
Installing /powerplant/workspace/cfltxm/miniconda3/envs/biocondabuilder/conda-bld/perl-module-build_1522805102380/_h_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placeh/lib/site_perl/5.26.0/Module/Build/Bundling.pod
Installing /powerplant/workspace/cfltxm/miniconda3/envs/biocondabuilder/conda-bld/perl-module-build_1522805102380/_h_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placeh/lib/site_perl/5.26.0/Module/Build/Platform/os2.pm
Installing /powerplant/workspace/cfltxm/miniconda3/envs/biocondabuilder/conda-bld/perl-module-build_1522805102380/_h_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placeh/lib/site_perl/5.26.0/Module/Build/Platform/Default.pm
Installing /powerplant/workspace/cfltxm/miniconda3/envs/biocondabuilder/conda-bld/perl-module-build_1522805102380/_h_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placeh/lib/site_perl/5.26.0/Module/Build/Platform/VOS.pm
Installing /powerplant/workspace/cfltxm/miniconda3/envs/biocondabuilder/conda-bld/perl-module-build_1522805102380/_h_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placeh/lib/site_perl/5.26.0/Module/Build/Platform/cygwin.pm
Installing /powerplant/workspace/cfltxm/miniconda3/envs/biocondabuilder/conda-bld/perl-module-build_1522805102380/_h_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placeh/lib/site_perl/5.26.0/Module/Build/Platform/Unix.pm
Installing /powerplant/workspace/cfltxm/miniconda3/envs/biocondabuilder/conda-bld/perl-module-build_1522805102380/_h_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placeh/lib/site_perl/5.26.0/Module/Build/Platform/aix.pm
Installing /powerplant/workspace/cfltxm/miniconda3/envs/biocondabuilder/conda-bld/perl-module-build_1522805102380/_h_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placeh/lib/site_perl/5.26.0/Module/Build/Platform/MacOS.pm
Installing /powerplant/workspace/cfltxm/miniconda3/envs/biocondabuilder/conda-bld/perl-module-build_1522805102380/_h_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placeh/lib/site_perl/5.26.0/Module/Build/Platform/darwin.pm
Installing /powerplant/workspace/cfltxm/miniconda3/envs/biocondabuilder/conda-bld/perl-module-build_1522805102380/_h_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placeh/lib/site_perl/5.26.0/Module/Build/Platform/VMS.pm
Installing /powerplant/workspace/cfltxm/miniconda3/envs/biocondabuilder/conda-bld/perl-module-build_1522805102380/_h_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placeh/lib/site_perl/5.26.0/Module/Build/Platform/Windows.pm
Installing /powerplant/workspace/cfltxm/miniconda3/envs/biocondabuilder/conda-bld/perl-module-build_1522805102380/_h_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placeh/man/man3/Module::Build::Platform::darwin.3
Installing /powerplant/workspace/cfltxm/miniconda3/envs/biocondabuilder/conda-bld/perl-module-build_1522805102380/_h_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placeh/man/man3/Module::Build::API.3
Installing /powerplant/workspace/cfltxm/miniconda3/envs/biocondabuilder/conda-bld/perl-module-build_1522805102380/_h_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placeh/man/man3/Module::Build::Platform::VMS.3
Installing /powerplant/workspace/cfltxm/miniconda3/envs/biocondabuilder/conda-bld/perl-module-build_1522805102380/_h_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placeh/man/man3/Module::Build::Platform::MacOS.3
Installing /powerplant/workspace/cfltxm/miniconda3/envs/biocondabuilder/conda-bld/perl-module-build_1522805102380/_h_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placeh/man/man3/Module::Build::Platform::aix.3
Installing /powerplant/workspace/cfltxm/miniconda3/envs/biocondabuilder/conda-bld/perl-module-build_1522805102380/_h_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placeh/man/man3/Module::Build::Compat.3
Installing /powerplant/workspace/cfltxm/miniconda3/envs/biocondabuilder/conda-bld/perl-module-build_1522805102380/_h_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placeh/man/man3/Module::Build::ConfigData.3
Installing /powerplant/workspace/cfltxm/miniconda3/envs/biocondabuilder/conda-bld/perl-module-build_1522805102380/_h_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placeh/man/man3/Module::Build::Platform::Default.3
Installing /powerplant/workspace/cfltxm/miniconda3/envs/biocondabuilder/conda-bld/perl-module-build_1522805102380/_h_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placeh/man/man3/Module::Build::Authoring.3
Installing /powerplant/workspace/cfltxm/miniconda3/envs/biocondabuilder/conda-bld/perl-module-build_1522805102380/_h_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placeh/man/man3/Module::Build::Platform::Unix.3
Installing /powerplant/workspace/cfltxm/miniconda3/envs/biocondabuilder/conda-bld/perl-module-build_1522805102380/_h_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placeh/man/man3/Module::Build::Platform::cygwin.3
Installing /powerplant/workspace/cfltxm/miniconda3/envs/biocondabuilder/conda-bld/perl-module-build_1522805102380/_h_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placeh/man/man3/Module::Build.3
Installing /powerplant/workspace/cfltxm/miniconda3/envs/biocondabuilder/conda-bld/perl-module-build_1522805102380/_h_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placeh/man/man3/Module::Build::Notes.3
Installing /powerplant/workspace/cfltxm/miniconda3/envs/biocondabuilder/conda-bld/perl-module-build_1522805102380/_h_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placeh/man/man3/Module::Build::PPMMaker.3
Installing /powerplant/workspace/cfltxm/miniconda3/envs/biocondabuilder/conda-bld/perl-module-build_1522805102380/_h_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placeh/man/man3/Module::Build::Platform::Windows.3
Installing /powerplant/workspace/cfltxm/miniconda3/envs/biocondabuilder/conda-bld/perl-module-build_1522805102380/_h_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placeh/man/man3/Module::Build::Base.3
Installing /powerplant/workspace/cfltxm/miniconda3/envs/biocondabuilder/conda-bld/perl-module-build_1522805102380/_h_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placeh/man/man3/Module::Build::Bundling.3
Installing /powerplant/workspace/cfltxm/miniconda3/envs/biocondabuilder/conda-bld/perl-module-build_1522805102380/_h_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placeh/man/man3/Module::Build::Platform::VOS.3
Installing /powerplant/workspace/cfltxm/miniconda3/envs/biocondabuilder/conda-bld/perl-module-build_1522805102380/_h_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placeh/man/man3/Module::Build::Platform::os2.3
Installing /powerplant/workspace/cfltxm/miniconda3/envs/biocondabuilder/conda-bld/perl-module-build_1522805102380/_h_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placeh/man/man3/Module::Build::Cookbook.3
Installing /powerplant/workspace/cfltxm/miniconda3/envs/biocondabuilder/conda-bld/perl-module-build_1522805102380/_h_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placeh/bin/config_data

Resource usage statistics from building perl-module-build:
   Process count: 5
   CPU time: Sys=0:00:01.2, User=0:00:04.5
   Memory: 67.6M
   Disk usage: 7.2K
   Time elapsed: 0:00:59.6

Packaging perl-module-build
INFO:conda_build.build:Packaging perl-module-build
Packaging perl-module-build-0.4224-pl526_0
INFO:conda_build.build:Packaging perl-module-build-0.4224-pl526_0
number of files: 46
Traceback (most recent call last):
  File "/workspace/cfltxm/miniconda3/envs/biocondabuilder/bin/conda-build", line 11, in <module>
    sys.exit(main())
  File "/workspace/cfltxm/miniconda3/envs/biocondabuilder/lib/python3.6/site-packages/conda_build/cli/main_build.py", line 420, in main
    execute(sys.argv[1:])
  File "/workspace/cfltxm/miniconda3/envs/biocondabuilder/lib/python3.6/site-packages/conda_build/cli/main_build.py", line 411, in execute
    verify=args.verify)
  File "/workspace/cfltxm/miniconda3/envs/biocondabuilder/lib/python3.6/site-packages/conda_build/api.py", line 199, in build
    notest=notest, need_source_download=need_source_download, variants=variants)
  File "/workspace/cfltxm/miniconda3/envs/biocondabuilder/lib/python3.6/site-packages/conda_build/build.py", line 2105, in build_tree
    notest=notest,
  File "/workspace/cfltxm/miniconda3/envs/biocondabuilder/lib/python3.6/site-packages/conda_build/build.py", line 1515, in build
    built_package = bundlers[output_d.get('type', 'conda')](output_d, m, env, stats)
  File "/workspace/cfltxm/miniconda3/envs/biocondabuilder/lib/python3.6/site-packages/conda_build/build.py", line 917, in bundle_conda
    files = post_process_files(metadata, initial_files)
  File "/workspace/cfltxm/miniconda3/envs/biocondabuilder/lib/python3.6/site-packages/conda_build/build.py", line 780, in post_process_files
    post_build(m, new_files, build_python=python)
  File "/workspace/cfltxm/miniconda3/envs/biocondabuilder/lib/python3.6/site-packages/conda_build/post.py", line 637, in post_build
    osx_is_app=osx_is_app)
  File "/workspace/cfltxm/miniconda3/envs/biocondabuilder/lib/python3.6/site-packages/conda_build/post.py", line 54, in fix_shebang
    with io.open(path, encoding=locale.getpreferredencoding(), mode='r+') as fi:
PermissionError: [Errno 13] Permission denied: '/workspace/cfltxm/miniconda3/envs/biocondabuilder/conda-bld/perl-module-build_1522805102380/_h_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placeh/bin/config_data'
```

**Comment by aguang** [comment]
I am getting the same error.

**Comment by nsoranzo** [comment]
I'm also hitting this in #9881, seems a conda-build problem on macOS?

**Comment by nsoranzo** [comment]
This should be fixed by conda-build >=3.14.0, which bioconda just started using.
